test/envoye2e/stackdriver_plugin/fake_stackdriver/cmd/Makefile.test_stackdriver.mk

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+  
+.PHONY: test_stackdriver.build_and_push
+
+MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+SD_PATH := $(dir $(MKFILE_PATH))
+
+test_stackdriver.build_and_push:
+	cd $(SD_PATH) && go build main.go
+	docker build $(SD_PATH) -t gcr.io/istio-testing/test-stackdriver:1.0
```

Comment:
This will publish the same image every post submit and overwrite the old one. Is this what we want ?

Reply:
Where is this image being used?

Reply:
I am planning to use it in istio/istio e2e test, or any places that tests istio/sd integration. Yeah, I don't think it is necessary to version this image yet: it is just for testing, we don't really care about tracking the diffs of every commits.

Reply:
Can we not just publish it on an adhoc basis, instead every post submit. We can still have the code here. 
If you feel it has to be done this way, then I am fine.

Reply:
Sure, removed from postsubmit. Done.